src/main/java/duke/Duke.java

```diff
@@ -27,7 +29,7 @@ public Duke(String filePath) {
     }
 
     /**
-     * Instantiate a Duke object and run it.
+     * Instantiate a duke.Duke object and run it.
```

Comment:
To be particular, I noticed the Javadoc comments not following this excerpt from the code style page: "method header comments, the first sentence should start in the form Returns ..., Sends ..., Adds ... (not Return or Returnning etc.)".

It's not that serious though haha.